templates/navigation.hbs

```diff
@@ -1,11 +1,11 @@
     <div class="nav-container">
       <div class="container">
-        <div class="pure-menu pure-menu-horizontal">
+        <div class="pure-menu pure-menu-horizontal" role="navigation" aria-label="Main navigation">
           <form action="/releases/search" method="GET" class="landing-search-form-nav">
             {{#unless varsb.show_search_form}}
-            <input class="search-input-nav" name="query" type="text" placeholder="Find crate"{{#if varss.search_query}} value="{{varss.search_query}}"{{/if}}>
+            <input class="search-input-nav" name="query" type="text" aria-label="Find crate by search query" placeholder="Find crate"{{#if varss.search_query}} value="{{varss.search_query}}"{{/if}}>
```

Comment:
The sentence here doesn't sound very nice... What about "Input to search crate"?

Reply:
"Input to search crate," isn't exactly clear English. "Find crate by search query," more adequately describes what a user can expect when submitting the form. "Find [a] crate by search query," is possibly more accurate, but there's no guarantee the query will return only a single result, so I left it open-ended.

According to the axe-core specification (see my later comment for more information) the ARIA label attribute can contain more text than the element itself, but should adequately explain what a screen reading user will expect [when the form is submitted](https://dequeuniversity.com/rules/axe/3.2/label-content-name-mismatch). 

I certainly believe there's ways to improve the language here, but I don't favor this suggestion because what I've written more closely aligns with the spec (and passes validation).